<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QMainWindow" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="regular">
     <widget class="QPushButton" name="pushButton_up">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-45</number>
      </property>
      <property name="maximum">
       <number>45</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_input">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_set">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_right">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_down">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_left">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Regular Mode</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tank">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tank Mode</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio45">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>45</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio90">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio0">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tank_down">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tank_up">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="custom">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Custom Mode</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="front_left_servo_box">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="middle_left_servo_box">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="back_left_servo_box">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="back_right_servo_box">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="front_right_servo_box">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="middle_right_servo_box">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_submit_servo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit Servo Positions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_left_up">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_left_up_down">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↑↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_left_down">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_middle_down_down">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↓↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_right_down">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_right_down_up">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↓↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_right_up">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_middle_up_up">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↑↑</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Regular</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_tank">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tank</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_custom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_speed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>238</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>radioButton_custom</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_tank</zorder>
   <zorder>horizontalSlider_speed</zorder>
   <zorder>pushButton_submit</zorder>
   <zorder>label_4</zorder>
   <zorder>speed_label</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
